src/fastoad/gui/analysis_and_plots.py

```diff
@@ -17,15 +17,28 @@
 from typing import Dict
 
 import numpy as np
-import plotly
 import plotly.graph_objects as go
 from openmdao.utils.units import convert_units
 from plotly.subplots import make_subplots
 
 from fastoad.io import VariableIO
 from fastoad.openmdao.variables import VariableList
 
-COLS = plotly.colors.DEFAULT_PLOTLY_COLORS
+# This color list is an attempt to mimic the default color list (the one used
+# when no color is specified). And strangely enough, it is not
+# plotly.colors.DEFAULT_PLOTLY_COLORS
+COLS = [
+    "mediumblue",
+    "red",
+    "mediumseagreen",
+    "mediumpurple",
+    "darkorange",
+    "cyan",
+    "orangered",
+    "lightgreen",
+    "pink",
+    "orange",
+]
```

Comment:
Thanks for the attempt to mimic the colors. I think I found something even closer!

```suggestion
import plotly.express as px
COLS = px.colors.qualitative.Plotly
```
![newplot-6](https://user-images.githubusercontent.com/5703129/181244571-71473f97-2cb4-49e6-a4cf-012906c4bb39.PNG)

Reply:
Well done. That's the one!
Fixed in 6be69ae4ef542cb8dc25c6a0cd165d9642025f2e